src/ops/ops.go

```diff
@@ -917,3 +929,166 @@ func installerArgs(ignitionPath string, device string, extra []string) []string
 	}
 	return append(allArgs, device)
 }
+
+func (o *ops) getPointedIgnitionSource(ignitionPath string) (string, error) {
+	b, err := os.ReadFile(ignitionPath)
+	if err != nil {
+		return "", errors.Wrapf(err, "failed to read ignition file %s", ignitionPath)
+	}
+	conf, _, err := config_latest.ParseCompatibleVersion(b)
+	if err != nil {
+		return "", errors.Wrapf(err, "failed to parse ignition file %s", ignitionPath)
+	}
+	source := ""
+	for i := range conf.Ignition.Config.Merge {
```

Comment:
Wouldn't this will fail in case of https://github.com/openshift/assisted-installer/blob/2e590cfa9d7052cbc3cdf601b167dcda4a348502/src/installer/installer.go#L130?

Reply:
This is not used for SNO.  Please see `getEmbeddedIgnition`

Reply:
Tested it with SNO. 
SNO has a merge section, but the source is not a pointer to another ignition.  The ignition is embedded as base64 encoding.

Reply:
Here it is merged fro SNO
https://github.com/openshift/assisted-installer/blob/2e590cfa9d7052cbc3cdf601b167dcda4a348502/src/installer/installer.go#L236